docs/source/notebooks/Introduction to Ablator.ipynb

```diff
@@ -0,0 +1,163 @@
+    "\n",
+    "HPO involves systematically searching for the best set of hyperparameters that lead to optimal model performance. The objective is to find the hyperparameter configuration that helps the model reach the best metrics (e.g minimum loss value, or maximum accuracy) among all possible training attempts.\n",
+    "\n",
+    " \n",
+    "\n",
+    "To read more refer - [Hyperparameter Optimization](https://en.wikipedia.org/wiki/Hyperparameter_optimization)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c9c29d9",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Ablator's Role: Enhancing Ablation Studies and Hyperparameter Optimization\n",
+    "\n",
+    "As machine learning models grow in complexity, the number of components that need to be ablated also increases.This consequently expands the search space of possible configurations, requiring an efficient approach to horizontally scale multiple parallel experimental trials.Ablator is a  tool that aids in the horizontal scaling of experimental trials.\n",
+    "\n",
+    "Instead of manually configuring and conducting multiple experiments with various hyperparameter settings, ABLATOR automates this process.  It initializes experiments based on different hyperparameter configurations, tracks the state of each experiment, and provides experiment persistence. \n",
+    "\n",
+    "Ablator employs a random search algorithm when conducting ablation studies.On the other hand, for Hyperparameter Optimization (HPO), Ablator utilizes a greedy search strategy known as Tree-structured Parzen Estimators (TPE). This approach aims to efficiently fine-tune the parameters of a model by iteratively selecting values that maximize performance, leading to improved overall results.\n",
+    "\n",
+    "A few advantages of using Ablator are - \n",
+    "\n",
+    "- It is a tool that simplifies the process of prototyping and training machine learning models. It streamlines model experimentation and evaluation.\n",
+    "- It offers a flexible configuration system.\n",
+    "- It facilitates result interpretation through visualization.\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea7d5a28",
+   "metadata": {},
+   "source": [
+    "## Inside Ablator: Exploring its Modules and How They Work Together\n",
+    "\n",
+    "This section introduces the core modules within Ablator. We'll cover the Configuration Module, Training Module, Experiment Result Metrics Module, and Analysis Module. These modules collectively contribute to Ablator's seamless experiment management and insightful analysis capabilities. \n",
+    "\n",
+    "[**Configuration Module -**](https://ablator-tutorials.readthedocs.io/en/v0.0.1-misc/notebooks/Configuration-Basics.html)\n",
```

Comment:
Should point to the files in the same folder instead of a static link
```suggestion
    "[**Configuration Module -**](./Configuration-Basics.ipynb)\n",
```

Reply:
Also make it level-3 section (do the same to other modules)